respy/tests/resources/kw_97_base.yaml

```diff
@@ -1,14 +1,14 @@
 choices:
   edu:
     max: 20
-    start: [7]
-    lagged: [1]
-    share: [1]
+    start: [7, 8, 9, 10, 11]
+    lagged: [1, 1, 1, 1, 1]
```

Comment:
Do we want to stick with this for now? It is fine with me until we get better information from @bekauf. However, I made better experiences with setting lagged to 0 for everyone with less than 10 years.

Reply:
Changed it.